test/File/_files/magic.mime

```diff
@@ -1055,7 +1055,7 @@
 >0	byte		x		GameCube movie,
 >0x34	ubeshort	x		%d x
 >0x36	ubeshort	x		%d,
->0x26	ubeshort	x		%dµs,
+>0x26	ubeshort	x		%dÂµs,
```

Comment:
@shahariaazam what is the change here and in the other lines below?

Reply:
This is simply a more up-to-date magic.mime file.